src/Identity/IdentityServer/ApiClient.cs

```diff
@@ -33,7 +33,13 @@ public ApiClient(
         }
         else if (id == "desktop")
         {
-            RedirectUris = new[] { "bitwarden://sso-callback" };
+            var desktopUris = new List<string>();
+            desktopUris.Add("bitwarden://sso-callback");
+            for (var port = 8065; port <= 8070; port++)
+            {
+                desktopUris.Add(string.Format("http://localhost:{0}", port));
+            }
+            RedirectUris = desktopUris;
```

Comment:
Can we add a comment here explaining why we need to allow localhost as a redirect uri?